provisioning/Samples/service/EnrollmentGroupSample/Program.cs

```diff
@@ -1,34 +1,44 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using CommandLine;
 using System;
 using System.Security.Cryptography.X509Certificates;
 
 namespace Microsoft.Azure.Devices.Provisioning.Service.Samples
 {
     public class Program
     {
-        // The Provisioning Service connection string. This is available under the "Shared access policies" in the Azure portal.
-
-        // For this sample either:
-        // - pass this value as a command-prompt argument
-        // - set the PROVISIONING_CONNECTION_STRING environment variable 
-        // - create a launchSettings.json (see launchSettings.json.template) containing the variable
+        /// <summary>

```

Comment:
I think this xml comment doesn't belong here
maybe at the class level?

Reply:
I am referring to our other samples, like this [one](https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/main/iot-hub/Samples/device/MethodSample/Program.cs)

Do we have a standard regarding this?

Reply:
It's currently commenting the s_connectionString, move it to line 19 and you should be good

Reply:
corrected